### PR TITLE
Dev/154 155

### LINKED ISSUE #154: Different code quality into IGN Tag Historian than DM one
Hi, 
it seems the datas/points pushed into Ignition TagHistorian after a sync from DM performed by HMS module, do not provide the correct Quality. I.e. if from Ewon a tag is pushed to DM with good quality value ( 3 ) ( it's a memory tag ) it's stored into IGN historian with Quality = 0. It prevent some objects to work properly, i.e. Easy Chart.

### LINKED ISSUE #155: Boolean tag data type can not change value
With V2.0 and Ignition 8.1 can not change the value of a boolean tag.  Setup a string, int and boolean memory tag on a Flexy unit.  Was able to change the value of the string and int tags but not the boolean tag.

V1.2 and Ignition 8.1 you can change the value of a boolean data type tag.

### PR DESCRIPTION
**Pull request by it-hms**
closes #154 
closes #155

### CODE REVIEW

**Comment by it-hms** [comment]
@alexjhawk, These changes still need to be tested

File: eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/requests/m2w/M2WebEwonUpdateTagValuesRequest.java [hunk 1]
```diff
@@ -79,6 +79,27 @@ public M2WebEwonUpdateTagValuesRequest(
...
   }
 
+  /**
+   * Special conversion for tag values to string. Booleans must be converted to 1 or 0. Null values
+   * are converted to empty strings.
+   *
+   * @param obj - Tag value object that should be converted to a string
+   * @return string representation of the tag value
+   */
+  public static String TagValToString(Object obj) {
```
Comment (alexjhawk) [inline eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/requests/m2w/M2WebEwonUpdateTagValuesRequest.java:89]:
```suggestion
  public static String tagValToString(Object obj) {
```

File: eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/requests/m2w/M2WebEwonUpdateTagValuesRequest.java [hunk 2]
```diff
@@ -115,7 +136,7 @@ private static String getUpdateTagValuesRequestPath(
           .append("TagValue")
           .append(tagIndex)
           .append("=")
-          .append(tagNameValuePair.getValue().toString());
+          .append(TagValToString(tagNameValuePair.getValue()));
```
Comment (alexjhawk) [inline eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/requests/m2w/M2WebEwonUpdateTagValuesRequest.java:139]:
```suggestion
          .append(tagValToString(tagNameValuePair.getValue()));
```

**Review by alexjhawk — CHANGES_REQUESTED** [review]
It looks good, aside from the two recommendations. We can talk about testing tomorrow.

**Comment by benlbaum** [comment]
@it-hms @alexjhawk are we able to get v2.0.1 merged so that we can test the Issue #154 fix?

**Comment by it-hms** [comment]
@alexjhawk ,
The changes you requested are completed. Could you please add a review?

**Review by alexjhawk — APPROVED** [review]

**Pull request merged**